envisage/tests/test_core_plugin.py

```diff
@@ -34,11 +46,17 @@ class TestApplication(Application):
 class CorePluginTestCase(unittest.TestCase):
     """ Tests for the core plugin. """
 
+    def setUp(self):
+        push_on_trait_change_handler(reraise_exceptions=True)
+        push_observe_handler(reraise_exceptions=True)
+
+    def tearDown(self):
+        pop_observe_handler()
```

Comment:
Nothing here uses `observe`, so we can remove the push and pop for the observe handleres.
In fact, is any of these `push_*` and `pop_*` required for reproducing the error if we remove the fix?

Reply:
Ah, yes I believe they can all be removed now